src/fragments/lib/auth/js/switch-auth.mdx

```diff
@@ -49,62 +49,164 @@ For more information about working with Lambda Triggers for custom authenticatio
 
 To initiate a custom authentication flow in your app, call `signIn` without a password. A custom challenge needs to be answered using the `sendCustomChallengeAnswer` method:
 
-```javascript
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { CognitoUser } from '@aws-amplify/auth';
```

Comment:
@devnotfound, we are considering importing CognitoUser directly from the library. Can you remove this line and any references to this type as well?